src/python/pants/backend/debian/rules.py

```diff
@@ -1,7 +1,11 @@
 # Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
```

Comment:
Excellent. The inconsistent use of this across files was annoying me. I'm glad this will be fixed.

Reply:
Yeah, it bombs out a few things without it - I might add a fixer or linter to check for it down the road. We use > 3.9 things so often, may as well always have it.

Reply:
Some files I've tried to add this annotation to breaks.. just as a heads up it might not always be easy to just slap it on.

Reply:
👍🏽  Good to know - I've never seen that happen, but thanks for the heads up. 

I'm trying to avoid too much cleaning in files to keep the PRs even slightly readable, but man, this one and rules annoy me so much visually for some reason 🤦🏽